mock-etcd/src/mock_etcd.rs

```diff
@@ -225,14 +225,15 @@ impl MockEtcdInner {
     }
 
     /// Send watch response for a specific watch id to etcd client
+    /// - return true if send success
```

Comment:
Do we want to know the detail error if it happens?

Reply:
Not sure about how many kinds of  detail errors. Maybe we just add a log?
But it's necessary to know the sending is  success or fail.